exporter/containerimage/writer.go

```diff
@@ -214,10 +239,74 @@ func (ic *ImageWriter) exportLayers(ctx context.Context, refCfg cacheconfig.RefC
 	return out, err
 }
 
+func (ic *ImageWriter) exportAttestations(ctx context.Context, s session.Group, desc *ocispecs.Descriptor, attestations ...exporter.Attestation) ([]intoto.Statement, error) {
+	span, ctx := tracing.StartSpan(ctx, "export attestations")
+
+	eg, ctx := errgroup.WithContext(ctx)
```

Comment:
Where is this `eg` stuff used?

Reply:
My bad, this function was meant to process attestations in parallel :smile: